tests/integration/test_data/go_generate_packages.yaml

```diff
@@ -673,6 +673,207 @@ go_generate:
     - "pkg:golang/unicode%2Futf8"
     - "pkg:golang/unsafe"
     purl: "pkg:golang/github.com%2Fcachito-testing%2Fgo-generate@v0.0.0-20211021002400-45a57f36d5e8"
+  sbom:
+  - name: bytes
+    type: library
+    version: ""
+    purl: pkg:golang/bytes
```

Comment:
Not sure how I feel about empty string as the version for stdlib dependencies :thinking: 

The cycloneDX schema doesn't allow `null`, but the version key is not required. Maybe we should not include it at all for dependencies that don't have a version?

```shell
sed '/version: ""/d' --in-place tests/integration/test_data/*.yaml
```

Reply:
we should certainly include all dependencies

reason why I am passing there empty string, instead of omitting version, is for consumers of sbom, as they actually might be expecting that version there ......

but if you insist that cachito should do this, I can do that in osbs .....

Reply:
It would be incorrect to require version since version is not a required key

Reply:
removed empty versions